### PR TITLE
Rise2 spinedem 1023 return default error from sandbox

### PR DESCRIPTION
**Pull request by richardseacome1-nhs**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: sandbox/src/services/patient-searcher.js [hunk 1]
```diff
@@ -64,11 +64,17 @@ function buildPatientResponse(examplePatients = [], searchScore = 1.0) {
                 resource: patient,
             })
         });
+        return response;
     } else {
-        delete response.entry
-    }
+        return {
```
Comment (richardcoates5-nhs) [inline sandbox/src/services/patient-searcher.js:69]: Do we have any coding standards for indentation of nested dictionaries?

I can't see anything from a quick look but I think this style is hiding a lot of detail which can be shown with a different style

`{
    resourceType: "OperationOutcome",
    timestamp: datefns.format(Date.now(), "yyyy-MM-dd'T'HH:mm:ss+00:00"),
    "issue": [
        {
            "severity": "warning",
            "code": "not-supported", 
            "details": {
                "coding": [
                    {
                        "system": "https://fhir.nhs.uk/R4/CodeSystem/Spine-ErrorOrWarningCode", 
                        "version": "1",
                        "code": "not-supported",
                        "display": "Unsupported operation"
                    }
                ]
            }, 
            "diagnostics": "This mock endpoint has no example response for this combination of search parameters"
        }
    ]
}`

(I'm not suggesting that style is the one that should be used)

File: sandbox/src/services/patient-searcher.js [hunk 2]
```diff
@@ -64,11 +64,17 @@ function buildPatientResponse(examplePatients = [], searchScore = 1.0) {
...
+            resourceType: "OperationOutcome",
+            timestamp: datefns.format(Date.now(), "yyyy-MM-dd'T'HH:mm:ss+00:00"),
+            "issue": [{"severity": "warning", "code": "not-supported",
```
Comment (richardcoates5-nhs) [inline sandbox/src/services/patient-searcher.js:72]: Do the keys need to have quotation marks around them?

File: sandbox/src/services/patient-searcher.js [hunk 3]
```diff
@@ -64,11 +64,17 @@ function buildPatientResponse(examplePatients = [], searchScore = 1.0) {
...
-        delete response.entry
-    }
+        return {
+            resourceType: "OperationOutcome",
+            timestamp: datefns.format(Date.now(), "yyyy-MM-dd'T'HH:mm:ss+00:00"),
+            "issue": [{"severity": "warning", "code": "not-supported", 
+            "details": {"coding": [{"system": "https://fhir.nhs.uk/R4/CodeSystem/Spine-ErrorOrWarningCode", 
+            "version": "1", "code": "not-supported", "display": "Unsupported operation"}]}, 
+            "diagnostics": "This mock endpoint has no example response for this combination of search parameters"}]}
+        }
```
Comment (richardcoates5-nhs) [inline sandbox/src/services/patient-searcher.js:76]: This final `}` is very misleading as it's for the else but lined up with the return. Whatever happens with the rest of the formatting I think this should be one indentation less

File: sandbox/src/services/patient-searcher.js (hunk 1 above)
Comment (richardseacome1-nhs) [reply sandbox/src/services/patient-searcher.js:69]: That's using the established template for an error message, there is no scope to change it in this ticket - but certainly it's something that could be reviewed/raised under tech debt if needed.

Comment (richardseacome1-nhs) [reply sandbox/src/services/patient-searcher.js:69]: converted to boom.notImplemented error.

File: sandbox/src/services/patient-searcher.js (hunk 2 above)
Comment (richardseacome1-nhs) [reply sandbox/src/services/patient-searcher.js:72]: converted to boom.notImplemented error.

**Review by richardcoates5-nhs — APPROVED** [review]

**Pull request merged**